src/diffusers/configuration_utils.py

```diff
@@ -552,9 +565,12 @@ def extract_init_dict(cls, config_dict, **kwargs):
         return init_dict, unused_kwargs, hidden_config_dict
 
     @classmethod
-    def _dict_from_json_file(cls, json_file: Union[str, os.PathLike]):
-        with open(json_file, "r", encoding="utf-8") as reader:
-            text = reader.read()
+    def _dict_from_json_file(cls, json_file: Union[str, os.PathLike], dduf_reader=None):
```

Comment:
Do we wanna annotate the type hint of `dduf_reader`? No biggie, though.

Reply:
yeah I will add the annotation !